packages/bundle/src/adaptiveCards/Attachment/CommonCard.js

```diff
@@ -39,5 +39,6 @@ CommonCard.propTypes = {
     content: PropTypes.shape({
       tap: PropTypes.any
     }).isRequired
-  }).isRequired
+  }).isRequired,
+  styleOptions: PropTypes.any.isRequired
```

Comment:
I think `CommonCard` should use `connectToWebChat` to obtain `styleSet.options` directly.